source/FileSystemDsc.psd1

```diff
@@ -1,48 +1,41 @@
 @{
+    RootModule           = 'FileSystemDsc.psm1'
+
     # Version number of this module.
-    moduleVersion      = '0.0.1'
+    ModuleVersion        = '0.0.1'
 
     # ID used to uniquely identify this module
-    GUID               = '86a20a80-3bcd-477e-9b90-ec8d52fbe415'
+    GUID                 = '86a20a80-3bcd-477e-9b90-ec8d52fbe415'
+
+    CompatiblePSEditions = @('Core', 'Desktop')
 
     # Author of this module
-    Author             = 'DSC Community'
+    Author               = 'DSC Community'
 
     # Company or vendor of this module
-    CompanyName        = 'DSC Community'
+    CompanyName          = 'DSC Community'
 
     # Copyright statement for this module
-    Copyright          = 'Copyright the DSC Community contributors. All rights reserved.'
+    Copyright            = 'Copyright the DSC Community contributors. All rights reserved.'
 
     # Description of the functionality provided by this module
-    Description        = 'This module contains DSC resources for managing file systems.'
+    Description          = 'This module contains DSC resources for managing file systems.'
 
     # Minimum version of the Windows PowerShell engine required by this module
-    PowerShellVersion  = '5.1'
+    PowerShellVersion    = '5.1'
 
     # Minimum version of the common language runtime (CLR) required by this module
-    CLRVersion         = '4.0'
-
-    # Functions to export from this module
-    FunctionsToExport  = @()
-
-    # Cmdlets to export from this module
-    CmdletsToExport    = @()
-
-    # Variables to export from this module
-    VariablesToExport  = @()
-
-    # Aliases to export from this module
-    AliasesToExport    = @()
```

Comment:
We should only remove the property `CmdletsToExport` (for it to show up in PowerShell), the other properties should remain.